var/spack/repos/builtin/packages/py-tensorflow/package.py

```diff
@@ -24,6 +48,7 @@ class PyTensorflow(Package, CudaPackage, ROCmPackage, PythonExtension):
     version("2.15.1", sha256="f36416d831f06fe866e149c7cd752da410a11178b01ff5620e9f265511ed57cf")
     version("2.15.0", sha256="9cec5acb0ecf2d47b16891f8bc5bc6fbfdffe1700bdadc0d9ebe27ea34f0c220")
     version("2.14.1", sha256="6b31ed347ed7a03c45b906aa41628ac91c3db7c84cb816971400d470e58ba494")
+    version("2.14.0-rocm-enhanced", branch="r2.14-rocm-enhanced-nohipblaslt-build")
```

Comment:
Is there a tarball available for this? Branches are moving targets and can't be checksummed.

Reply:
No tarball is currently available for this. I will look into it.